src/config.cpp

```diff
@@ -13,6 +13,7 @@ Config::Config(bool has_cmd, bool bt_verbose) : bt_verbose_(bt_verbose)
   config_map_ = {
     { ConfigKeyBool::cpp_demangle, { .value = true } },
     { ConfigKeyBool::lazy_symbolication, { .value = false } },
+    { ConfigKeyString::license, { .value = std::nullopt } },
```

Comment:
This is a bit strange to me. The default license is GPL but we set the default config value to *none* here and only return "GPL" from `BPFtrace::get_license`. Why not set it to "GPL" here and only override the value from SPDX/config, if specified?

If the only reason is that we don't have a config priority defined for config value coming from a script comment header (SPDX here), then I believe that we should introduce that. It will also save the introduction of `std::optional<T> Config::try_get`.

Reply:
Yes. Thank you @viktormalik for articulating this. I also didn't understand why we didn't want to set the default in the config.